blackbox/docs/appendices/release-notes/3.2.1.rst

```diff
@@ -0,0 +1,70 @@
+.. _version_3.2.1:
+
+=============
+Version 3.2.1
+=============
+
+Released on 2019/01/14.
+
+.. NOTE::
+
+    If you are upgrading a cluster, you must be running CrateDB 2.0.4 or higher
+    before you upgrade to 3.2.1.
+
+    We recommend that you upgrade to the latest 3.1 release before moving to
+    3.2.1.
+
+    You cannot perform a `rolling upgrade`_ to this version. Any upgrade to this
+    version will require a `full restart upgrade`_.
+
+    When restarting, CrateDB will migrate indexes to a newer format. Depending
+    on the amount of data, this may delay node start-up time.
+
+    Please consult the :ref:`version_3.0.0_upgrade_notes` before upgrading.
+
+.. WARNING::
+
+    Tables that were created prior to upgrading to CrateDB 2.x will not
+    function with 3.2 and must be recreated before moving to 3.2.x.
+
+    You can recreate tables using ``COPY TO`` and ``COPY FROM`` while running a
+    2.x release into a new table, or by `inserting the data into a new table`_.
+
+    Before upgrading, you should `back up your data`_.
+
+.. _rolling upgrade: http://crate.io/docs/crate/guide/best_practices/rolling_upgrade.html
+.. _full restart upgrade: http://crate.io/docs/crate/guide/best_practices/full_restart_upgrade.html
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
+.. _inserting the data into a new table: https://crate.io/docs/crate/reference/en/latest/admin/system-information.html#tables-need-to-be-recreated
+
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+Changelog
+=========
+
+Fixes
+-----
+
+- Fixed a ``NullPointerException`` that could occur using ``array_difference``.
+
+- Fixed a race condition that could lead to stuck queries. One case this could
```

Comment:
is this the exhaustive list of cases? or just an example? I would like to suggest this bullet be rewritten but I don't know how to do that without knowing this